src/infobar.cpp

```diff
@@ -447,19 +446,6 @@ class w_cpu : public gwidget
                            procview->cpu_times(1, Proc::CPUTIME_USER),
                            procview->old_cpu_times(0, Proc::CPUTIME_USER));
                 }
-
-#ifdef LINUX
-                int z = drawSPECTRUM2(p, mw, gheight + 1 + r * (gheight - 1),
-                                      buff, total, user, system, nice,
-                                      gheight - 2, bar_w);
-#endif
-
-#ifdef SOLARIS
-                int z = drawSPECTRUM2(p, mw, gheight + 1 + r * (gheight - 1),
-                                      buff, total, user, system, wait,
-                                      gheight - 2, bar_w);
-// drawSPECTRUM(p,0,cpu_id*10,buff,total,user,system,wait);
-#endif
```

Comment:
Please don't remove the above instances of `drawSPECTRUM2`. They draw a part of the info bar. I think your goal was the removal of the unused variable `int z`.

Reply:
> Please don't remove the above instances of `drawSPECTRUM2`. They draw a part of the info bar. I think your goal was the removal of the unused variable `int z`.

Thanks @tsujan. That was a mistake.